index.html

```diff
@@ -15,24 +15,26 @@
     <link href="https://fonts.googleapis.com/css?family=Lato:100&display=swap" rel="stylesheet"> 
 </head>
 <body>
+    <!--Nav-->
     <nav>
         <div id="nav-container">
```

Comment:
#### `REQUIRED`

Use a valid CSS gradient property. `-moz-linear-gradient` is an unknown vendor extension (for browsers other than Mozilla) in the corresponding CSS file. See below  

https://github.com/geraldgsh/apple-page-clone/blob/b520102d45d11ea707df719f627c9fe0069f65fc/style.css#L38

##### Suggestion: Add this too for other browsers (Chrome).

```css
    background-image: linear-gradient(#828282 0%, #484848 50%, #383838 51%, #575757 100%);
```

Reply:
@bolah2009 Noted with many thanks.